Source/Core/Core/HW/GCPadEmu.cpp

```diff
@@ -50,7 +50,10 @@ GCPad::GCPad(const unsigned int index) : m_index(index)
   // buttons
   groups.emplace_back(m_buttons = new ControllerEmu::Buttons(_trans("Buttons")));
   for (unsigned int i = 0; i < sizeof(named_buttons) / sizeof(*named_buttons); ++i)
-    m_buttons->controls.emplace_back(new ControllerEmu::Input(named_buttons[i]));
+  {
+    const std::string& ui_name = (named_buttons[i] == "Start") ? "START" : named_buttons[i];
```

Comment:
Err, sorry, missed the part about the INI compatibility in the quote above; that should be included too IMO. I just want to avoid future devs changing the original array (as you did before amending) or doing a general "lets upper-case this string" in here, breaking things that weren't obvious.